### PR TITLE
Add some text format

### PR DESCRIPTION
**Pull request by Ajnasz**
Make comments italic and make spell error colors red
Make javascript arrow functions yellow, just like the function keyword.

### CODE REVIEW

File: colors/yowish.vim [hunk 1]
```diff
@@ -51,7 +51,7 @@ call s:Hi('String'    , 'NONE' , s:color.lightGreen  , 'NONE')
 hi! link Character String
 " Default {{{1
 call s:Hi('ColorColumn'  , s:color.backgroundDark   , s:color.yellow          , 'bold')
-call s:Hi('Comment'      , 'NONE'                   , s:color.comment         , 'NONE')
+call s:Hi('Comment'      , 'NONE'                   , s:color.comment         , 'italic')
```
Comment (KabbAmine) [inline colors/yowish.vim]: A lot of people are not fan of italic comments (myself included :smile:), so IMO it should be optional via an option.

Better to add `g:yowish.comment_italic` in `autoload/yowish.vim` file, then check it here:

```vim
if g:yowish.comment_italic
    call s:Hi..... 'italic'
else
    call s:Hi..... 'NONE'
```

File: colors/yowish.vim [hunk 2]
```diff
@@ -77,7 +77,7 @@ call s:Hi('Question'     , 'NONE'                   , s:color.lightGreen      ,
 call s:Hi('Search'       , s:color.yellow           , s:color.backgroundDark  , 'bold')
 call s:Hi('SpecialKey'   , 'NONE'                   , s:color.backgroundLight , 'NONE')
 call s:Hi('Special'      , 'NONE'                   , s:color.textLight       , 'NONE')
-call s:Hi('SpellBad'     , 'NONE'                   , 'NONE'                  , 'undercurl')
+call s:Hi('SpellBad'     , 'NONE'                   , s:color.red             , 'undercurl')
```
Comment (KabbAmine) [inline colors/yowish.vim]: No a fan here either, but if you need it please make it also optional via an option as explained in my previous comment.

**Comment by KabbAmine** [comment]
Thank you for your contribution, please see my comments.
If you add those changes as an options, please document them in the README.

The js arrow commit is good and will be merged after fixing and force pushing the 1st commit.

**Comment by Ajnasz** [comment]
Thank you for your feedbacks!
I added the requested changes.
Instead of setting the `spellBad` color to red I made it configurable, so one can choose a color from the palette and also added a helper function, so similar configuration can be added easily.

**Comment by KabbAmine** [comment]
LGTM :+1:

**Pull request merged**